frontend/src/container/IngestionSettings/MultiIngestionSettings.tsx

```diff
@@ -561,10 +561,11 @@ function MultiIngestionSettings(): JSX.Element {
 					APIKey.created_at,
 					formatTimezoneAdjustedTimestamp,
 				);
-				const formattedDateAndTime =
-					APIKey &&
-					APIKey?.expires_at &&
-					getFormattedTime(APIKey?.expires_at, formatTimezoneAdjustedTimestamp);
+
+				const expiresOn =
+					!APIKey?.expires_at || APIKey?.expires_at === '0001-01-01T00:00:00Z'
```

Comment:
this looks like a very specific hard-coded handling, don't we have any generic way, also when and why do this timestamp comes.

Reply:
initially I wanted to add a condition similar to APIKeys i.e. https://github.com/SigNoz/signoz/blob/f80c58e5afaa4e6191021487d6140135a734cc65/frontend/src/container/APIKeys/APIKeys.tsx#L351-L355

But @grandwizard28 mentioned that for `expires_at` of **Ingestion settings**, we don't get 0 value since this is coming as a datetime string. For **No expiry** cases, we get the zero value date `0001-01-01T00:00:00Z`. therefore, checked against the zero date.

Thread link: https://signoz-team.slack.com/archives/C02TJ466H8U/p1735126709621099

Reply:
Ok, 

@grandwizard28 - just wondering if it safe to assume that always this zero value will come, or should we at UI add and handle more cases here?

Rest looks good, approved